src/missing-types.d.ts

```diff
@@ -312,10 +315,11 @@ declare module '@higlass/utils' {
     import type { ScaleContinuousNumeric } from 'd3-scale';
     import type { TilesetInfo } from '@higlass/services';
 
-    type ChromInfo<Name extends string> = {
+    export type ChromInfo<Name extends string = string> = {
```

Comment:
I think we can reuse this in our assembly definitions as well:

https://github.com/gosling-lang/gosling.js/blob/db1b657f3918272759236c1317c0ab00fc4acfb2/src/core/utils/assembly.ts#L12

Reply:
Ok great!